### PR TITLE
T165872: add no ha option for huwiki

### PR DESCRIPTION
**Pull request by Pix1234**

### CODE REVIEW

File: editquality/feature_lists/huwiki.py [hunk 1]
```diff
@@ -17,7 +17,8 @@
     hungarian.informals.revision.diff.match_delta_decrease,
     hungarian.informals.revision.diff.match_prop_delta_sum,
     hungarian.informals.revision.diff.match_prop_delta_increase,
-    hungarian.informals.revision.diff.match_prop_delta_decrease
+    hungarian.informals.revision.diff.match_prop_delta_decrease,
+    informals_no_ha = english.informals.excluding(["ha"])
```
Comment (Ladsgroup) [inline editquality/feature_lists/huwiki.py]: This will end up as syntax error, please move it outside of definition of the list.

**Review by Ladsgroup — CHANGES_REQUESTED** [review]

**Comment by Ladsgroup** [comment]
Also it would be great to reduce number of the commits, I can squash them to one commit when merging too though. As you prefer.

**Review by Ladsgroup — APPROVED** [review]

**Comment by Ladsgroup** [comment]
I squashed it as ten commits for this doesn't make sense.

**Comment by halfak** [comment]
:P  wasn't supposed to merge until we have a new model

**Pull request merged**